.travis.yml

```diff
@@ -12,10 +12,25 @@ addons:
 cache:
   directories:
     - node_modules
-services:
-  #- docker
 before_script:
-  #Docker setup goes here
   - pwd
   - cp ~/build/HackIllinois/api-2017/config/test.config.template ~/build/HackIllinois/api-2017/config/test.config
```

Comment:
Is this line necessary if we're deploying to production? I see a production config is getting copied over at the end, but would that mean we have both a prod and a test config installed?